content/participate/design.adoc

```diff
@@ -0,0 +1,22 @@
+---
+layout: project
+title: Art and Design
+section: participate
+tags:
+  - outreach-programs
+  - designer
+---
+
+
+== Contribute
+
+Jenkins Area Meetups love special Jenkins art for their meetup, and some plugins are important enough to deserve their own logo. You can check the link:/artwork/[gallery] for some amazing contributions, or propose your own on the link:https://github.com/jenkins-infra/jenkins.io[website repository].
```

Comment:
Would be nice to link https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#adding-a-logo below. We will put detailed guidelines there later